src/routes/[shortUrl]/stats/+page.svelte

```diff
@@ -0,0 +1,70 @@
+<script context="module" lang="ts">
+	import { error } from '@sveltejs/kit';
+
+	export async function load({ params }) {
+		const { shortUrl } = params;
+		const kv = (globalThis as any)[process.env.VITE_URLS] as KVNamespace;
+
+		// Fetch the transition data from KV
+		const transitionData = await kv.get(`${shortUrl}:transition`, { type: 'json' });
```

Comment:
Saving list of events in a blob JSON seems a bit unmaintainable. Why not to use cursor to get paginated output?